admin/pre_reg/utils.py

```diff
@@ -0,0 +1,12 @@
+import itertools
+
+from django.contrib.auth import models
+
+def get_prereg_reviewers():
+    return itertools.chain(
+        ((None, 'None'), ),
```

Comment:
I would move the prepending (None, 'None') into the place(s) this method is being called just for readability. 

An alternative to the comprehension would be 
```python
from django.db.models.functions import Concat
from django.db.models import Value

MyUser.objects
    .filter(groups__name='prereg_group')
    .annotate(fuller_name=Concat('first_name', Value(' '), 'last_name'))
    .values_list('email', 'fuller_name')
``` 
which looks up all the users with a group of prereg_group and returns a list of tuples of their `('email', 'full_name')`. I called it fuller_name because django doesn't allow you to annotate onto an object with a property of the same name.

Ends up making this sql:

```sql
SELECT "common_auth_myuser"."email", COALESCE("common_auth_myuser"."first_name", %s) || COALESCE(COALESCE(%s, %s) || COALESCE("common_auth_myuser"."last_name", %s), %s) AS "fuller_name" FROM "common_auth_myuser" INNER JOIN "common_auth_myuser_groups" ON ( "common_auth_myuser"."id" = "common_auth_myuser_groups"."myuser_id" ) INNER JOIN "auth_group" ON ( "common_auth_myuser_groups"."group_id" = "auth_group"."id" ) WHERE "auth_group"."name" = %s ORDER BY "common_auth_myuser"."email" ASC LIMIT 21' - PARAMS = (u'', u' ', u'', u'', u'', u'prereg_group'); args=('', ' ', '', '', '', 'prereg_group')
```